pom.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.hotels</groupId>
+    <artifactId>hotels-oss-parent</artifactId>
+    <version>2.0.5</version>
+  </parent>
+
+  <groupId>com.hotels</groupId>
+  <artifactId>housekeeping</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
```

Comment:
Need to add `<inceptionYear>2016</inceptionYear>` here

Reply:
It will be easier to keep this as 2016 since all the license headers in the files copied over from CT require this. The alternative is to put this to 2017 and then add a section in the pom to configure it to ignore all the files pulled over from CT and those files will have to be kept up to date manually going forward -> more work than its worth.